src/pulik/atm/machines/TerminalATM.java

```diff
@@ -3,29 +3,27 @@
 import pulik.atm.cards.Card;
 
 public class TerminalATM extends TellerMachine {
-	
+
 	@Override
 	public void insert(Card card) {
 		System.out.println("inserting\n" + card);
 		super.insert(card);
 	}
-	
+
 	public void eject() {
 		System.out.println("ejecting\n" + insertedCard);// todo
 		super.eject();
 
 	}
-	
-	public void inputPin(String pin) {
-		boolean correctPin = insertedCard.checkPin(pin); 
-		authorised = correctPin;
-		if (correctPin) {
-			System.out.println("Authorised"); // todo
-		} else {
-			System.out.println("Denied"); // todo
+
+	@Override
+	public void showBalance() {
+		try {
+			System.out.println(insertedCard.getBalance());
+		} catch (Exception e) {
+			System.out.println("Sorry, incorrect pin");
 		}
+
 	}
-	
-	
 
 }
```

Comment:
Notice the red symbol on GitHub on this line - it means that the new line symbol at the end of a file is missing.

It is common practice to end a file with a new line. Please add it here.